sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -1248,10 +1333,15 @@ def set_index(s):
 
   rename_axis = frame_base._elementwise_method('rename_axis', base=pd.Series)
 
+  @frame_base.with_docs_from(pd.Series, name='is_unique')
   @frame_base.args_to_kwargs(pd.Series)
   @frame_base.populate_defaults(pd.Series)
   @frame_base.maybe_inplace
   def replace(self, to_replace, value, limit, method, **kwargs):
+    """`method` is not supported in the Beam DataFrame API because it is
+    order-sensitive, it must not be specified.
```

Comment:
"order-sensitive, it must not be specified." -> "order-sensitive. It cannot be specified."

Reply:
Done